_sass/_base.scss

```diff
@@ -4,7 +4,7 @@
 body, h1, h2, h3, h4, h5, h6,
 p, blockquote, pre, hr,
 dl, dd, ol, ul, figure {
-  margin: 0;
+  //margin: 0;
```

Comment:
This is a pretty big global change and usualy a standard reset/normalize move.  I'd actually prefer to keep this.  Not sure why we would comment it out.  Is there some backstory I'm missing.